hyppo/conditional_independence/FCIT.py

```diff
@@ -0,0 +1,237 @@
+import time
+import joblib
+
+import numpy as np
+from scipy.stats import ttest_1samp
+from sklearn.metrics import mean_squared_error as mse
+from sklearn.preprocessing import StandardScaler
+
+from sklearn.model_selection import GridSearchCV
+from sklearn.model_selection import ShuffleSplit
+from sklearn.tree import DecisionTreeRegressor
+
+from .base import ConditionalIndependenceTest, ConditionalIndependenceTestOutput
+
+
+class FCIT(ConditionalIndependenceTest):
+    r"""
+    Fast Conditional Independence test statistic and p-value
+
+    The Fast Conditional Independence Test is a non-parametric
+    conditional independence test :footcite:p:`chalupka2018FastConditionalIndependence`.
+
+    Parameters
+    ----------
+    model: Sklearn regressor
+        Regressor used to predict input data :math: `Y`
+    cv_grid: dict
+       Dictionary of parameters to cross-validate over when training regressor.
+    num_perm: int
+        Number of data permutations to estimate the p-value from marginal stats.
+    prop_test: float
+        Proportion of data to evaluate test stat on.
+    discrete: tuple of string
+        Whether :math: `X` or :math: `Y` are discrete
+    Notes
+    -----
+    The motivation for the test rests on the assumption that if :math: `X \not\!\perp\!\!\!\perp Y \mid Z`,
```

Comment:
This is not rendering correctly in the documentation. See: https://deploy-preview-315--hyppo.netlify.app/api/generated/hyppo.conditional_independence.fcit#hyppo.conditional_independence.FCIT